lib/src/components/NxList/NxListLinkItem.tsx

```diff
@@ -34,7 +34,8 @@ const NxListLinkItem = forwardRef<HTMLLIElement, NxListLinkItemProps>(
         <li ref={ref} className={liClassNames} {...attrs} aria-selected={selected} aria-current={selected}>
           <a aria-disabled={includesDisabledClass(aClassNames)}
              className={aClassNames}
-             href={href}
+             href={disabled ? undefined : href}
+             role={disabled ? 'link' : undefined}
```

Comment:
For a11y specified in [ARIA in HTML](https://www.w3.org/TR/html-aria/#docconformance), to disable <a> link
-  the link should not have href attr
-  have aria-disabled set to true
-  add a role="link" so that it is considered a link by screen readers